rootfs/api/models/app.py

```diff
@@ -329,7 +329,7 @@ def _scale_pods(self, scale_types):
                 log_event(self, err, logging.ERROR)
                 raise
 
-    def deploy(self, user, release):
+    def deploy(self, release):
```

Comment:
This didn't use the `user` arg? Crufty goodness.

(If this and the below changes are an unrelated cleanup, they should ideally be a separate commit and a separate PR though.)